File: generator/output/entityModel/LanguageModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Language
*/
@Data()
public class LanguageModel {
  public Long id = null;

  /**
  *
  */
  public String tag = null;

  /**
  *
  */
  public Integer deleted = null;

  /**
  *
  */
  public String description = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}

File: generator/output/ts-model/language.model.ts
class LanguageModel {
  public id: number = 0;

  // tag 
  public tag: string;

  // deleted 
  public deleted: boolean = false;

  // description 
  public description: string;

  public created_at: string;

  public updated_at: string;
}

const LanguageModelFields = {
  id: "id",
  tag: "tag",
  deleted: "deleted",
  description: "description",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {LanguageModel, LanguageModelFields};
